src/entry_points/gentest.py

```diff
@@ -0,0 +1,303 @@
+        "$TEST_COMMENT", "gentest autogenerated test for tx.hash " + tr.tr_hash[2:]
+    )
+
+    # Print a nice .py storage pre
+    pad = "            "
+    state_str = ""
+    for address, account in state.items():
+        if isinstance(account, dict):
+            account_obj = Account(**account)
+            state_str += '        "' + str(address) + '": Account(\n'
+            state_str += pad + "balance=" + str(account_obj.balance) + ",\n"
+            if address == tr.transaction.sender:
+                state_str += pad + "nonce=" + str(tr.transaction.nonce) + ",\n"
+            else:
+                state_str += pad + "nonce=" + str(account_obj.nonce) + ",\n"
+
+            if account_obj.code is None:
+                state_str += pad + 'code="0x",\n'
+            else:
+                state_str += pad + 'code="' + str(account_obj.code) + '",\n'
+            state_str += pad + "storage={\n"
+
+            if account_obj.storage is not None:
+                for record, value in account_obj.storage.items():
+                    state_str += pad + '    "' + str(record) + '" : "' + str(value) + '",\n'
+
+            state_str += pad + "}\n"
+            state_str += "        ),\n"
+    test = test.replace("$PRE", state_str)
+
+    # Print legacy transaction in .py
+    tr_str = ""
+    tr_str += pad + "ty=" + str(tr.transaction.ty) + ",\n"
+    tr_str += pad + "chain_id=" + str(tr.transaction.chain_id) + ",\n"
+    tr_str += pad + "nonce=" + str(tr.transaction.nonce) + ",\n"
+    tr_str += pad + 'to="' + str(tr.transaction.to) + '",\n'
+    tr_str += pad + "gas_price=" + str(tr.transaction.gas_price) + ",\n"
+    tr_str += pad + "protected=False,\n"
+    tr_str += pad + 'data="' + str(tr.transaction.data) + '",\n'
+    tr_str += pad + "gas_limit=" + str(tr.transaction.gas_limit) + ",\n"
+    tr_str += pad + "value=" + str(tr.transaction.value) + ",\n"
+    tr_str += pad + "v=" + str(tr.transaction.v) + ",\n"
+    tr_str += pad + "r=" + str(tr.transaction.r) + ",\n"
+    tr_str += pad + "s=" + str(tr.transaction.s) + ",\n"
+
+    test = test.replace("$TR", tr_str)
+    return test
+
+
+class PyspecConfig:
+    """
+    Main class to manage Pyspec config
+    """
+
+    @dataclass
+    class RemoteNode:
+        """
+        Remote node structure
+        """
+
+        name: str
+        node_url: str
+        client_id: str
+        secret: str
+
+    remote_nodes: List["PyspecConfig.RemoteNode"]
+
+    def __init__(self, config_path: str):
+        """
+        Initialize pyspec config from file
+        """
+        with open(config_path, "r") as file:
+            data = json.load(file)
+            self.remote_nodes = [self._json_to_remote_node(node) for node in data["remote_nodes"]]
+
+    def _json_to_remote_node(self, d):
+        return PyspecConfig.RemoteNode(
+            name=d["name"],
+            node_url=d["node_url"],
+            client_id=d["client_id"],
+            secret=d["secret"],
+        )
```

Comment:
```suggestion
            self.remote_nodes = [PyspecConfig.RemoteNode(**node) for node in data["remote_nodes"]]
```
Small python trick here: double asterisk passes the dictionary contents as named parameters to the function call.